### PR TITLE
Add license

### LINKED ISSUE #41: Ajouter license

### PR DESCRIPTION
**Pull request by HAEKADI**
closes #41

### CODE REVIEW

**Review by geoffreyaldebert — COMMENTED** [review]
I usually implement MIT but if you think it's better ok for me. Let @XavierJp decide

**Comment by XavierJp** [comment]
@geoffreyaldebert I dont have strong opinion on this. We started using AGPLv3 since Patrick recommended it but entreprise.data uses MIT

**Comment by bzg** [comment]
Permissive licenses (MIT, Apache 2.0, BSD*) should be considered first. In this case, MIT seems fine.

See https://guide-juridique-logiciel-libre.etalab.gouv.fr/#8

**Comment by XavierJp** [comment]
@bzg we used AGPLv3 on many repos. Should we migrate them ? Is there a protocol for License migration ?
 
https://github.com/betagouv/api.gouv.fr
https://github.com/etalab/annuaire-entreprises-site
https://github.com/betagouv/datapass

**Review by XavierJp — APPROVED** [review]

**Comment by bzg** [comment]
If DINUM is the sole owner of the code, there is no special procedure for switching to a new license.

If there are no reason for choosing a restrictive license over a permissive one, yes, you should switch to a permissive license.

You also need to check that there are no licensing constraints due to the licenses of the modules you require: see [this page](https://guide-juridique-logiciel-libre.etalab.gouv.fr/#7).

Let me know if I can help further!

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2022 Etalab
```
Comment (bzg) [inline LICENSE:3]: s/Etalab/Direction interministérielle du numérique

Comment (HAEKADI) [reply LICENSE:3]: Okay. Donc je mets les deux ?

**Comment by XavierJp** [comment]
@bzg do you mean that to use MIT licenses, you need every thrid party libs you use to use MIT ?

**Comment by bzg** [comment]
To publish your code under a _permissive license_ (such as MIT, Apache 2.0 or BSD*), yes, the third-party libraries you use needs to be under a permissive license.  If one of the third-party license is not a permissive one, let's discuss, because the legal constraints really depend on the way your new code depends on the libraries.

File: LICENSE (hunk 1 above)
Comment (bzg) [reply LICENSE]: Oui, pardon, les deux c'est bien.

**Pull request merged**